### PR TITLE
fix(storage): add key resource to HMAC methods that require it

### PR DESCRIPTION
**Pull request by BrennaEpp**

### CODE REVIEW

File: storage/v1/retry_tests.json [hunk 1]
```diff
@@ -23,11 +23,11 @@
...
         {"name": "storage.buckets.lockRetentionPolicy",   "resources": ["BUCKET"]},
-        {"name": "storage.buckets.testIamPermission",     "resources": ["BUCKET"]},
+        {"name": "storage.buckets.testIamPermissions",     "resources": ["BUCKET"]},
         {"name": "storage.default_object_acl.get",        "resources": ["BUCKET"]},
         {"name": "storage.default_object_acl.list",       "resources": ["BUCKET"]},
-        {"name": "storage.hmacKey.delete",                "resources": []},
-        {"name": "storage.hmacKey.get",                   "resources": []},
+        {"name": "storage.hmacKey.delete",                "resources": ["HMAC_KEY"]},
+        {"name": "storage.hmacKey.get",                   "resources": ["HMAC_KEY"]},
         {"name": "storage.hmacKey.list",                  "resources": []},
```
Comment (cojenco) [inline storage/v1/retry_tests.json:31]:
```suggestion
        {"name": "storage.hmacKey.list",                  "resources": ["HMAC_KEY"]},
```

Comment (cojenco) [reply storage/v1/retry_tests.json:31]: Thanks for adding this! Could we also add the "HMAC_KEY" resource to the hmacKey operations: `list` `update` and also [lines 152-155](https://github.com/googleapis/conformance-tests/blob/37ddc58e5c7ebc2847e67e1cba39bdee9e86e260/storage/v1/retry_tests.json#L152-L155)?

Comment (BrennaEpp) [reply storage/v1/retry_tests.json]: good catch!

**Review by cojenco — APPROVED** [review]
LGTM, thanks!

**Pull request merged**